std_msgs/Header header

int8[] gates

geometry_msgs/PoseArray gate_poses
